lib/http/2/compressor.rb

```diff
@@ -103,6 +103,7 @@ def initialize(**options)
         @table = []
         @options = default_options.merge(options)
         @limit = @options[:table_size]
+        @dynamic_table_size_update_limit = options[:settings_header_table_size] || 4096
```

Comment:
Why do we need this if @limit is already tracking this?

Reply:
my bad, I confused myself originally, then I realized it was set properly for the limit already